public/locales/en/status.json

```diff
@@ -10,6 +10,7 @@
   "countMore": "...and {count} more",
   "StatusConnected": {
     "repoSize": "Hosting {repoSize} of data",
+    "shareSize": "Downloaded {downloadedSize} and Shared {sharedSize} ({ratio})",
```

Comment:
@SgtPooki @hacdias thoughts on presentation / phrasing here? 

These are stats for current session and not the total across history.

Same for "Discovered peers". 


Maybe we should prefix them with "Current session" or "Since {startup date}"?